modo/api.py

```diff
@@ -170,6 +190,13 @@ def remove_element(self, element_id: str):
             data_file = self.path / attrs["data_path"]
             if data_file.exists():
                 data_file.unlink()
+            elif isinstance(
+                self.archive.store, zarr.storage.FSStore
+            ) and self.archive.store.fs.exists(data_file):
+                self.archive.store.fs.rm(str(data_file))
```

Comment:
**suggestion**: removal of files happens silently, would it make sense to have some messages to either confirm that a file was deleted or found? I guess the only way currently is to check the bucket directly, so a message could help.

Reply:
Fair point, added a message (after deletion) to **modo.api** for clarification in 1ab950d699cf0756bb89e019fa56da187c611281 and a prompt asking for confirmation to the **cli** in 1f4f3e94d8641d4404636be7591565cd3fef7e4a